README.md

```diff
@@ -15,6 +15,61 @@
 Parley provides an API for implementing rich text layout.
 It is backed by [Swash](https://github.com/dfrg/swash).
 
+## The Parley text stack
+
+Currently, Parley directly depends on four crates: Fontique, Swash, Skrifa, and Peniko.
+These crates cover different pieces of the text-rendering process.
+
+### Peniko
+
+Peniko builds on top of [kurbo](https://crates.io/crates/kurbo) and provides vocabulary types for 2D rendering.
+
+Peniko mostly just exports types like `Blob`, `Color`, `Gradient`, `Brush`, `Point`, `Rect`, `Vec2`, etc.
+
+### Fontique
+
+Fontique provides font enumeration and fallback.
+
+**Font enumeration** means listing (enumerating) all the fonts installed on the system.
+It also means collecting metadata about those fonts: whether they are serif, sans-serif, monospace, their weight, the code points they cover, etc.
+The library will also load the full font into memory as required using mmap.
```

Comment:
This is not how mmap works. This should probably be something like “The library will use memory-mapped IO , which allows portions to be loaded into memory once and shared between processes on the system.”.. or maybe no attempt should be made to explain mmap.

Reply:
👍 to not mentioning `mmap`

Reply:
There is interest in the Office Hours call to at least mention here that the fonts are memory mapped rather than loaded wholesale.

Reply:
How about this:

> The library will also load the full font into memory as required, using memory-mapped IO for efficiency.

Reply:
That also is misleading, because part of the point is that it doesn't "load the full font into memory". If done correctly, there will be portions of the font that never get loaded into memory.
They are mapped, not loaded.

Reply:
I can give a sentence that is accurate and good, if you tell me what's wrong with what I suggested.

Reply:
No, I think your sentence works. I thought it was maybe too specific, but I don't really understand the stack enough to judge.

Reply:
The crucial/interesting thing about mmap in this circumstance is that it is the norm for accessing system fonts.
That's why I favored getting rid of this sentence: if you don't already know what you're doing with font technology, it's hard to understand why this is important; but if you do know about this stuff then it says something obvious.
The feeling on office hours is that we should mention it, so I'm okay with that but it should be accurate.

Reply:
I think the current version is accurate?

Reply:
It's good. :shipit: